src/reader.py

```diff
@@ -245,3 +336,22 @@ def EstimateCovariance(DataX, DataY, SysLength = {}, SysStrength = {}, ScaleX =
                 Matrix[x, y] = Matrix[x, y] + DataX["Data"]["yerr"]["sys"][x][IX] * DataY["Data"]["yerr"]["sys"][y][IY] * Factor
 
     return Matrix
+
+
+class tuneData:
```

Comment:
A bit trivial since it's not really respected in those code, but python convention is to capitalize a class. I think it's still helping to try to follow as much as possible:
```suggestion
class TuneData:
```